src/server_manager/install_scripts/install_server.sh

```diff
@@ -168,17 +248,43 @@ function start_shadowbox() {
     -e "SB_METRICS_URL=${SB_METRICS_URL:-}"
     -e "SB_DEFAULT_SERVER_NAME=${SB_DEFAULT_SERVER_NAME:-}"
   )
-  docker run -d "${docker_shadowbox_flags[@]}" "${SB_IMAGE}" >/dev/null
+  local readonly RUN_DOCKER="docker run -d ${docker_shadowbox_flags[@]} ${SB_IMAGE}"
+  local readonly STDERR_OUTPUT
+  STDERR_OUTPUT=$(sg docker -c "$RUN_DOCKER 2>&1 >/dev/null")
+  local readonly RET=$?
+  if [[ $RET -eq 0 ]]; then
+    return 0
+  fi
+  log_error "FAILED"
+  if [[ $STDERR_OUTPUT = *"Conflict"* ]]; then
+    handle_docker_container_conflict shadowbox
+  else
+    log_error "$STDERR_OUTPUT"
+    return 1
+  fi
 }
 
 function start_watchtower() {
   # Start watchtower to automatically fetch docker image updates.
   # Set watchtower to refresh every 30 seconds if a custom SB_IMAGE is used (for
   # testing).  Otherwise refresh every hour.
-  readonly WATCHTOWER_REFRESH_SECONDS="${WATCHTOWER_REFRESH_SECONDS:-3600}"
+  local WATCHTOWER_REFRESH_SECONDS="${WATCHTOWER_REFRESH_SECONDS:-3600}"
   declare -a docker_watchtower_flags=(--name watchtower --restart=always)
   docker_watchtower_flags+=(-v /var/run/docker.sock:/var/run/docker.sock)
-  docker run -d "${docker_watchtower_flags[@]}" v2tec/watchtower --cleanup --tlsverify --interval $WATCHTOWER_REFRESH_SECONDS >/dev/null
+  local readonly RUN_DOCKER="docker run -d ${docker_watchtower_flags[@]} v2tec/watchtower --cleanup --tlsverify --interval $WATCHTOWER_REFRESH_SECONDS"
+  local readonly STDERR_OUTPUT
+  STDERR_OUTPUT=$(sg docker -c "$RUN_DOCKER 2>&1 >/dev/null")
+  local readonly RET=$?
+  if [[ $RET -eq 0 ]]; then
+    return 0
+  fi
+  log_error "FAILED"
+  if [[ $STDERR_OUTPUT = *"Conflict"* ]]; then
```

Comment:
Rather than processing the stderr, which is fragile, let's check if the containers are running.

Reply:
Done in d281454.